pages/common/stern.md

```diff
@@ -0,0 +1,28 @@
+# stern
+
+> Tail multiple pods and containers from Kubernetes.
+> More information: <https://github.com/wercker/stern/>.
+
+- Tail all pods within a current namespace:
+
+`stern .`
+
+- Tail all pods with status waiting:
+
+`stern . --container-state {{waiting}}`
```

Comment:
Is there a fixed set of statuses ? Then we can do `waiting|started|stopped` for eg. And change the description to "Tail all pods with a given status"

Reply:
Oh, I forgot the syntax for that. Check if it's ok now?